webviz_config/webviz_deprecated.py

```diff
@@ -0,0 +1,85 @@
+from typing import Dict, Union, Callable, Tuple, cast, Optional, List
+import inspect
+import warnings
+
+from ._plugin_abc import WebvizPluginABC
+from ._deprecation_store import (
+    DEPRECATION_STORE,
+    DeprecatedPlugin,
+    DeprecatedArgument,
+    DeprecatedArgumentCheck,
```

Comment:
These becomes "public" API, since you can do `webviz_config.webviz_deprecated.{DEPRECATION_STORE, DeprecatedPlugin, ...}`. Maybe we don't need to expose this? E.g. add leading `_` or something equivalent.